src/app/shared/components/frame/frame.component.html

```diff
@@ -0,0 +1,3 @@
+<section class="frame">
```

Comment:
Where is fhis class coming from? It should be defined in a component css or just assign the proper tailwind classes